app/controllers/chapters_controller.rb

```diff
@@ -57,7 +57,7 @@ def show
           @work.anonymous? ? ts("Anonymous") : @work.pseuds.sort.collect(&:byline).join(', '),
           @work.title + " - Chapter " + @chapter.position.to_s)
 
-      @kudos = @work.kudos.with_user.includes(:user).by_date
```

Comment:
Can the `by_date` scope be removed?

Reply:
Yeah, it's not used anymore. I removed it.